openpype/hosts/aftereffects/api/extension/js/main.js

```diff
@@ -156,7 +156,8 @@ function main(websocket_url){
         return runEvalScript("importFile('" + escapedPath +"', " +
                                          "'" + data.item_name + "'," +
                                          "'" + JSON.stringify(
-                                         data.import_options) + "')")
+                                         data.import_options) + "', " +
+                                         "'" + data.fps + "')")
```

Comment:
fps should be added in the `import_options`, not as a new argument of the function